index.md

```diff
@@ -29,7 +29,7 @@ This code is not exhaustive or complete. It serves to distill our common underst
 
 ### Diversity Statement
 
-We encourage everyone to participate and are committed to building a community for all. Although we may not be able to satisfy everyone, we all agree that everyone is equal. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
+We encourage everyone to participate and are committed to building a community for all. Although we may not be able to satisfy everyone, we all agree that everyone is worthy of respect. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
```

Comment:
@batjko, that would indeed be an improvement, though I still think 'equally worthy of respect', is simpler. Perhaps 'equally worthy of being treated with respect', then?

@Addvilz, 'equal and' phrasings are just as problematic.

@gyehuda, care to chime in at this point?